### ISSUE TITLE
Easier way to disable metaboxes

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Permissions#818]
Can we add an option in the admin area to disable metaboxes?

There should be an easier option than using wp-config.php

**Comment by agapetry** [comment]
@stevejburge I'm not sure the minimal use cases for this justify another item in Permissions > Settings.  We already have capabilities to hide them individually from non-Administrators. I would rather determine why Capabilities > Editor Features fails to remove these metaboxes, and fix that: 

![image](https://user-images.githubusercontent.com/43488774/208523213-5ba6ff5b-4b6c-4f83-bb53-4682d7e956c4.png)

**Comment by stevejburge** [comment]
@agapetry I guess the limitation with that limiting metaboxes is a Pro feature.

I'm not sure we should ask people to buy Capabilities Pro for this.

**Comment by stevejburge** [comment]
This came up again a couple of times this week from users. 

> I would like to hide the metaboxes everywhere - I only installed the plugin to control access to the media library

**Comment by stevejburge** [comment]
This has come up again: https://github.com/publishpress/PublishPress-Permissions/issues/877 Let's get this done

**Comment by stevejburge** [comment]
Idea for this: if you select a post type and taxonomy, two options appear underneath:

- Enable metabox
- Include on "Permissions" screen

**Comment by agapetry** [comment]
@stevejburge Can you confirm that a Permissions screen exclusion would suppress post types from the "Add New" tabs, not from displaying currently stored Specific Permissions and Supplemental Roles for that post type?

**Comment by agapetry** [comment]
@stevejburge This request would have been satisfied by fully disabling filtering for Post Types other than Media Library:

> I would like to hide the metaboxes everywhere - I only installed the plugin to control access to the media library

**Comment by stevejburge** [comment]
@agapetry Fair point here: "what percentage of users will want to enable filtering for a post type". That number will be small.

I'm aware of what I said earlier about hiding something that will be used by small percentage of users :)

My defense of this idea is that option would also act as an explanation of where users can configure permissions (Metabox, and Groups/Permissions) screen.

I'm open to other options.

**Comment by agapetry** [comment]
> My defense of this idea is that option would also act as an explanation of where users can configure permissions (Metabox, and Groups/Permissions) screen.

@stevejburge If the main concern is introducing plugin functionality to new users, I think that would be better accomplished with a welcome screen. Additional control(s) under each post type checkbox would be quite heavy (more than doubling the screen height for a potentially long list of post types).

**Comment by stevejburge** [comment]
@agapetry No, that is just a secondary benefit.

The main issue is customer feedback: they want some way an easy way to disable the metaboxes.
